packages/replay/rollup.bundle.config.js

```diff
@@ -0,0 +1,34 @@
+import commonjs from '@rollup/plugin-commonjs';
+import replace from '@rollup/plugin-replace';
+
+import { makeBaseBundleConfig, makeBundleConfigVariants } from '../../rollup/index.js';
+
+import pkg from './package.json';
+
+const baseBundleConfig = makeBaseBundleConfig({
+  bundleType: 'addon',
+  entrypoints: ['src/index.ts'],
+  jsVersion: 'es6',
+  licenseTitle: '@sentry/replay',
+  outputFileBase: () => 'bundles/replay',
+  packageSpecificConfig: {
+    plugins: [
+      replace({
+        preventAssignment: true,
+        values: {
+          __SENTRY_REPLAY_VERSION__: JSON.stringify(pkg.version),
+        },
+      }),
+      commonjs(),
+    ],
+    output: {
+      // set exports to 'named' or 'auto' so that rollup doesn't warn about
+      // the default export in `worker/worker.js`
+      exports: 'auto',
```

Comment:
Why did you choose `auto` over `named` here? (I forget why, but I know that in other places, I had to use `named`.)

Reply:
That was a leftover from CDN bundle initial experiments. Interestingly enough, I just noticed that we don't need to set any `exports` property so I removed it entirely. Apparently we just need this for the NPM package config.